docs/docs/widgets/pagination.md

```diff
@@ -0,0 +1,71 @@
+---
+id: pagination
+title: Pagination
+---
+# Pagination
+
+Pagination enables the user to select a specific page from a range of pages.
+
+:::tip
+It is used seperate content into discrete pages.
+:::
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - Widget Reference - Code editor](/img/widgets/pagination/pagination.png)
+
+</div>
```

Comment:
Let's use the `<img>` tags. We don't need to use markdown wrapped inside div anymore.